src/lib/mod.rs

```diff
@@ -195,5 +195,9 @@ mod version;
 
 /// Handles the command line arguments and executes the runner.
 pub fn run_cli(command_name: String, sub_command: bool) {
+    #[cfg(windows)]
+    if let Err(err) = ansi_term::enable_ansi_support() {
+        eprintln!("error enabling ANSI support: {:?}", err);
```

Comment:
not sure we need to print it to the users.... if it doesnt work, no big deal i guess. wdyt?
also, why not do it in makers.rs? since cargo is already fixing it for us, we can just push the fix to the makers executable only.

Reply:
I wrote it in `makers.rs` first but then I've moved it to CLI to satisfy linters (otherwise it breaks the rule `not used crate ansi_term` or something like that). 
I've added `eprintln` for easier user reporting when it fails.

So
I'll move it back to `makers.rs` and satisfy/disable the linter rule for it somehow.
Should I remove that `eprintln` call?

Reply:
i wonder how to silence that linter for specific case, so in future i don't forget some crates in...
maybe than, just add some bool from makers/main -> cli that tell it if to use it or not... crappy hack.

eprintln -> ya, lets remove it. i'm not sure users would find that helpful and many people already complain that cargo make prints out too much stuff.

Reply:
Code updated in the latest commit.

From https://doc.rust-lang.org/rustc/lints/listing/allowed-by-default.html#unused-crate-dependencies:
> This lint is "allow" by default because it can provide false positives depending on how the build system is configured. For example, when using Cargo, a "package" consists of multiple crates (such as a library and a binary), but the dependencies are defined for the package as a whole. If there is a dependency that is only used in the binary, but not the library, then the lint will be incorrectly issued in the library.